dipy/reconst/tests/test_dki.py

```diff
@@ -4,6 +4,7 @@
 import random
 import dipy.reconst.dki as dki
 import dipy.reconst.dti as dti
+from cvxpy import settings as s
```

Comment:
Thank you for this! We try to avoid 1 letter variable. The minimum is 3 letters when possible. 

I do not think you need `settings`. you can just do `import cvxpy` and then `cvxpy.ECOS` or `cvxpy.CLARABEL`.

Also, the default one was ECOS, why do you change it to Clarabel?  I recommend to keep ecos for now, so `cvpy_solver=cvxpy.ECOS` since the warning just tell us to be explicit. We might change the default solver when we know more about the difference between both

Reply:
Changed the import in 8f5988c. Using `cvxpy.ECOS` (or previously `s.CLARABEL`) results in the test failing: please check it locally. I assume that what the warning _only_ means is that ECOS is the current default solver, but it will be changed: despite ECOS being the default solver, if I'm not mistaken, CVXPY _may_ internally pick another one that best suits the problem. I ignore whether in its verbose mode it would tell us the solver that it has been using.

Reply:
This fails now: https://github.com/dipy/dipy/actions/runs/7158982614/job/19491729377#step:9:4498
```
FAILED reconst/tests/test_dki.py::test_dki_fits - AttributeError: module 'cvxpy' has no attribute 'CLARABEL'
```

However, it looks like we should be able to call it:
https://github.com/cvxpy/cvxpy/blob/f8e3b2c29da53712cad6b89314588f10c3a96afa/cvxpy/__init__.py#L66

Any intuition @skoudoro ?

Reply:
it fails with anaconda, so we need to check cvxpy version on Anaconda and when this change has been introduced, I will look into it

Reply:
We might need to update our minimal version for cvxpy due to this following information: https://www.cvxpy.org/updates/index.html#ecos-deprecation

Reply:
I plan to do 2 PRs to improve the minimum version management. After that, you will need to rebase and it should be ok.

The strategy is to skip the tests if we do not respect the minimum version

Reply:
a970b79 rebased on master.

Reply:
https://github.com/dipy/dipy/pull/2999#pullrequestreview-1778264214 Resolving.